client-side/src/stories/list/List.scss

```diff
@@ -0,0 +1,142 @@
+@import '../variables.scss';
+
+
+.wrapper {
+
+    background-color: #ffffff;
+    padding: 12px;
+    position: absolute;
+    right: 10px;
+    top: 50px;
+    width: 220px;
```

Comment:
please use % instead of px
and move all colors to variables
do it in all the file